egs/sprakbanken_swe/s5/local/sprak_run_5c.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# This is neural net training on top of adapted 40-dimensional features.
+# 
+
+. ./cmd.sh
+
+uid=$1       # Here $uid is the LM identifier, e.g '3g'
+test1=$2
+test2=$3
+
+
+ steps/nnet2/train_tanh.sh \
+   --mix-up 8000 \
+   --initial-learning-rate 0.01 --final-learning-rate 0.001 \
+   --num-hidden-layers 4 --hidden-layer-dim 1024 \
+   --cmd "$decode_cmd" \
+   data/train data/lang_test_4g exp/tri4a_ali exp/nnet5c || exit 1
```

Comment:
when I ran the scripts on my computer, locally, all cmd settings was set to "run.pl", which was an earlier version of sprakbanken cmd.sh default setting. Is having (as it is now) default "queue.pl xxx" going to mess things up? 

I never used gpu, only cpu, but should I add the gpu flag anyways?